README.md

```diff
@@ -173,11 +174,12 @@ Timestamps are compared as numbers, and hashes are compared as strings. Since st
 
 All messages must pass the following validations in addition to specific validations for the message type:
 
-1. `message.timestamp` is not more than 1 hour ahead of system time.
-2. `message.account` must be a known account number in the AccountRegistry.
-3. `signerPubKey` should be a valid [Root Signer or Delegate Signer](#45-signer-authorizations) for `message.account`
-4. `hashFn(serializeFn(message))` must match `envelope.hash`, where hashFn is a Blake2B function and serializeFn performs JSON canonicalization.
-5. `EdDSA_signature_verify(envelope.hash, envelope.signerPubKey, envelope.signature)` should pass.
+1. `message.timestamp` is not more than 1 hour ahead of system time. 
+2. `message.id` must be a known fid number in the FIR.
+3. `signerPubKey` should be a valid [Root Signer or Delegate Signer](#45-signer-authorizations) for `message.id`
+4. `hashFn(serializeFn(message))` must match `envelope.hash`, where hashFn is a Blake2B function and serializeFn performs JSON canonicalization. 
+5.  `EdDSA_signature_verify(envelope.hash, envelope.signerPubKey, envelope.signature)` should pass.
```

Comment:
Weird extra space after the number. Would running prettier on the file fix little things like this?

Reply:
i'm going to do a separate PR to add a commit hook linter